superset-frontend/src/preamble.ts

```diff
@@ -40,7 +40,7 @@ if (typeof window !== 'undefined') {
   if (bootstrapData.common && bootstrapData.common.language_pack) {
     const languagePack = bootstrapData.common.language_pack;
     configure({ languagePack });
-    moment.locale(bootstrapData.common.locale);
+    moment.locale(bootstrapData.common.moment_locale);
```

Comment:
AFAIK we don't load additional locales for `moment` so this is probably not needed. 

Let's leave this frontend change to another PR since there might be more places you need to update.

Reply:
@ktmud 
Thank you for reviewing my code!

It's used in many places on `superset-front` to format time. The required values in current `LANGUAGES` config and `moment.js`  are inconsistent, and I think using `flag` as a supplement should be  ok with `moment.js`. 
`LANGUAGES` config :
```python
LANGUAGES = {
    "en": {"flag": "us", "name": "English"},
    "es": {"flag": "es", "name": "Spanish"},
    "it": {"flag": "it", "name": "Italian"},
    "fr": {"flag": "fr", "name": "French"},
    "zh": {"flag": "cn", "name": "Chinese"},
    "ja": {"flag": "jp", "name": "Japanese"},
    "de": {"flag": "de", "name": "German"},
    "pt": {"flag": "pt", "name": "Portuguese"},
    "pt_BR": {"flag": "br", "name": "Brazilian Portuguese"},
    "ru": {"flag": "ru", "name": "Russian"},
    "ko": {"flag": "kr", "name": "Korean"},
}
```
moment.js locale:
https://github.com/moment/moment/tree/develop/locale

I  wanted to add a `moment_locale` attribute in `LANGUAGES` before, but I don't think it's necessary for the time being, but I think the existing value is enough, so I adjusted the existing value.

Reply:
Users can set `LANGUAGES` with any locale or custom flags, but locales supported by `moment` is fixed. I think it's easier if we have one Superset `locale` config value everywhere and only do the transformation for each library when calling that library. The supported locales are enumerable so the transformation should be fairly simple.

Either way, I'd recommend limit the scope of this PR to only change humanize for CURD list first. i18n is a complex topic, let's proceed with caution.